Index: ibus-libpinyin/configure.ac
===================================================================
--- ibus-libpinyin.orig/configure.ac	2013-09-30 20:51:06.417019450 +0800
+++ ibus-libpinyin/configure.ac	2013-09-30 21:37:18.426424115 +0800
@@ -170,7 +170,7 @@
 if test x"$has_lua_extension" = x"no";
 then
     PKG_CHECK_MODULES(LUA, [
-        lua5.1
+        lua5.2
     ], [],
     [enable_lua_extension=no]
     );
